packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -255,6 +256,10 @@ class VmServiceWrapper implements VmService {
           if (parent != null && parent.frameId != 0)
             CpuProfileData.parentIdKey: '$isolateId-${parent.frameId}',
         };
+
+        if (current.resolvedUrl != null && current.resolvedUrl!.isNotEmpty) {
+          resolvedUrlMap[current.resolvedUrl!] = null;
```

Comment:
hoist `current.resolvedUrl` up into a local variable before this if statement. Then use the local var wherever you are currently using `current.resolvedUrl`. This will allow you to get rid of all the null checks following the first `!= null`.